README.md

```diff
@@ -27,7 +27,7 @@ To run Helios on Ethereum, run the command below, replacing `$ETH_RPC_URL` with
 helios ethereum --execution-rpc $ETH_RPC_URL
 ```
 
-`$ETH_RPC_URL` must be a [supported Ethereum Execution API Provider](#supported-execution-api-providers) that provides the `eth_getProof` endpoint. Infura does not currently support this. We recommend using Alchemy.
+`$ETH_RPC_URL` must be a supported Ethereum Execution API Provider that provides the `eth_getProof` endpoint. Infura does not currently support this. We recommend using Alchemy.
```

Comment:
apparently Infura works now, so we should remove the reference to it not woking.